src/main/webapp/i18n/de/programmingExercise.json

```
@@ -306,6 +307,17 @@
                     "testType": "Type",
                     "passedPercent": "Bestanden %"
                 },
+                "feedbackAnalysis": {
+                    "titleHeader": "Feedback Analyse",
+                    "title": "Fehleranalyse für {{exerciseTitle}}",
+                    "occurrence": "Häufigkeit",
+                    "feedback": "Feedback",
+                    "task": "Aufgabe",
+                    "testcase": "Testfall",
+                    "errorCategory": "Fehlerkategorie",
+                    "totalItems": "Insgesamt {{count}} Elemente",
+                    "error": "Ein Fehler ist beim Laden des Feedback aufgetreten."
```

I don't know why but doesn't the following sound better?
```suggestion
                    "error": "Beim Laden des Feedback ist ein Fehler aufgetreten."
```